src/tests/JIT/Regression/JitBlue/Runtime_40444/Runtime_40444.cs

```diff
@@ -1,66 +1,154 @@
+
```

Comment:
Nit: Extra new line

Reply:
I would like to just leave this matching the existing test in master.